Cargo.toml

```diff
@@ -42,7 +42,7 @@ wasmtime-wasi     = { workspace = true }
 wasi-common       = { workspace = true }
 wasi-cap-std-sync = { workspace = true }
 wasmtime-provider = { version = "1.7.0", features = ["cache"] }
-picky = { version = "7.0.0-rc.5", default-features = false, features = [ "x509", "ec", "chrono_conversion" ] }
+picky = { version = "7.0.0-rc.6", default-features = false, features = [ "x509", "chrono_conversion" ] }
```

Comment:
Considering that the `ec` feature has been added in rc3, I guess is fine to remove it. 

https://lib.rs/crates/picky/versions

Reply:
yep, should have commented on the commit, rc6 now has elliptic curve by default.

Reply:
To keep documented what the `ec` was enabling: https://docs.rs/crate/picky/7.0.0-rc.5/features